tests/test_faster_rcnn.py

```diff
@@ -426,8 +434,3 @@ def test_faster_rcnn_predict(mock_create, config, dataset):
     image = torchvision.transforms.functional.to_tensor(image)
     result = estimator.predict(image)
     assert result == []
-
-
-def test_clean_dir():
```

Comment:
If we delete this, will the temp folder be there forever?

Reply:
temp folder gets deleted automatically after the tests execution

Reply:
I remember Sanjay talked about if we don't remove it, it would be still in memory. @sanjayuconn , can you elaborate on this?

Reply:
<img width="1259" alt="Screen Shot 2020-08-03 at 11 19 46 PM" src="https://user-images.githubusercontent.com/26328148/90065259-d97dcd80-dca0-11ea-8e5f-4ef53d7e1f80.png">
if we keep it then we get the following warning, which says its trying to delete the temp dir but its not present. @BlairLee

Reply:
https://docs.python.org/3.4/library/tempfile.html#tempfile.TemporaryDirectory

- We are creating: 
tmp_dir = tempfile.TemporaryDirectory()

- so it requires manual clean up.

if we create using below
with TemporaryDirectory() as temp_dir:
    # ... do something with temp_dir
#.... automatically cleaned up when context exited

Reply:
@sanjayuconn according to tests the checkpoint file created in `train_all_test` is required by `predict` and `load` tests. If we use context manager they'll not be able to use it.